tensorflow_serving/model_servers/http_server.cc

```diff
@@ -83,20 +83,11 @@ net_http::HTTPStatusCode ToHTTPStatusCode(const Status& status) {
 }
 
 void ProcessPrometheusRequest(PrometheusExporter* exporter,
-                              const PrometheusConfig& prometheus_config,
                               net_http::ServerRequestInterface* req) {
   std::vector<std::pair<string, string>> headers;
   headers.push_back({"Content-Type", "text/plain"});
   string output;
-  Status status;
-  // Check if url matches the path.
-  if (req->uri_path() != prometheus_config.path()) {
```

Comment:
I think this check is redundant.

Reply:
(firstly sorry for the delayed response!)

yes this check is redundant, but having it here -- allows us to check for
potential misrouting either due to a bug in HTTP server handler code,
or handler registration code (in CreateAndStartHttpSever() below). i'd
like to keep it for now. wdyt?

Reply:
I see. I changed to keep it.